src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -50,20 +50,24 @@
  * </li>
  * </ul>
  * <p>
- * To prevent {@code FooCheck} violations from being reported write:
- * </p>
- * <pre>
- * &#64;SuppressWarnings("foo") interface I { }
- * &#64;SuppressWarnings("foo") enum E { }
- * &#64;SuppressWarnings("foo") InputSuppressWarningsFilter() { }
- * </pre>
- * <p>
  * Some real check examples:
  * </p>
  * <p>
  * This will prevent from invocation of the MemberNameCheck:
  * </p>
  * <pre>
+ * &lt;module name=&quot;TreeWalker&quot;&gt;
+ * &lt;module name=&quot;MemberName&quot;/&gt;
+ * &lt;module name=&quot;SuppressWarningsHolder&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>
+ * To configure filter to suppress audit events for annotations add:
+ * </p>
```

Comment:
Add filter to xml config above. Xml config should be single block with all required for example modules .
In other words `To configure filter to suppress ..` should not exist.

Reply:
done